### DISCUSSION TITLE
Playback issue of CompressedVideo messages

### DISCUSSION THREAD

**Discussion by christian-nils** [foxglove/community#872] [Q&A]
Hi,

I have converted some video data from a GoPro camera to your new CompressedVideo message. I can play it with the image panel (after creating an account, and signing in). The issue is that after the first keyframe found, the video starts to play but very, very slowly. When I pause the playback, the video continues to be played.

It feels like the software tries to render all the frames but as the rendering takes too much time, it just creates a build-up in frames to render.

Shall I fill an issue?

Regards,

Christian-Nils

**Comment by defunctzombie** [comment]
Hi Christian - to debug this kind of issue we need the file you've recorded and are trying to play. Could you please share that and we'd be happy to look into it.

**Reply by christian-nils** [reply]
Hi Roman,

I cannot share this video because it contains personal data. But I can record some data next week and share that with you.

Thanks for looking into it,

Greetings,

CN

**Reply by snosenzo** [reply]
Also it would be helpful to send us your chrome GPU information.
To do this, in the chrome address bar type: `chrome://gpu`, then click "Download report"  and post the file here.
For clarity, I'm wondering if GPU hardware acceleration is enabled/available for your system.

**Reply by christian-nils** [reply]
[about-gpu-2023-11-13T15-51-51-186Z.txt](https://github.com/foxglove/community/files/13336522/about-gpu-2023-11-13T15-51-51-186Z.txt)

I was not at the office today, so I will only be able to collect the data on Wednesday. I am running the snap version BTW.
